ios/Gekidou/Sources/Gekidou/Networking/Networki+Error.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
```

Comment:
Maybe the name was accidentally spelled wrong:

`Network+Error.swift` vs `Network(i)+Error.swift`